src/components/common/SideMenu.js

```diff
@@ -100,11 +138,56 @@ class SideMenu extends React.Component {
         >
           {this.renderDrawerHeader()}
           <div className={classes.contentWrapper}>
-            {this.renderDescription()}
             <LineSelector />
-            <OscillateSwitch />
-            <AmplitudeSlider disabled={!shouldOscillate} />
-            <FrequencyAdjuster />
+            <div className={classes.switchContainer}>
```

Comment:
I really like the factoring out of the switches and sliders. 👍

Reply:
👍